scripts/data_migrations/2018-10-31_migrate-data-sources.py

```diff
@@ -7,7 +7,7 @@
 from application import db
 from application.config import Config
 from application.factory import create_app
-from application.cms.models import Page, DataSource
+from application.cms.models import MeasureVersion, DataSource
```

Comment:
This feels like a really weird thing to have to do.

I guess it's a result of the original migration referencing things that it shouldn't have done?

These data migrations are historical... can we just leave them be or does that make things fail to start?  Should we delete them, or archive them to somewhere outside of the application? That way we don't need to maintain them indefinitely.

Reply:
Test migrations fail - whenever you reset the test db, _all_ migrations are executed. So these fail. It's a result of bad migrations in the first place. We should probably remove these/fix them to have the state inline. Maybe now is the time?

Reply:
Yeah it is weird. I don't immediately remember why I did it now - but something complained. Possibly it was just black, and I should skip it for this file...